packages/notebook/src/browser/notebook-editor-widget.tsx

```diff
@@ -81,6 +81,15 @@ export class NotebookEditorWidget extends ReactWidget implements Navigatable, Sa
     protected readonly onDidChangeModelEmitter = new Emitter<void>();
     readonly onDidChangeModel = this.onDidChangeModelEmitter.event;
 
+    protected readonly onPostKernelMessageEmitter = new Emitter<unknown>();
+    readonly onPostKernelMessage = this.onPostKernelMessageEmitter.event;
+
+    protected readonly onPostRendererMessageEmitter = new Emitter<{ rendererId: string; message: unknown }>();
```

Comment:
Suggestion: This should be its own type
```suggestion
    protected readonly onPostRendererMessageEmitter = new Emitter<RendererMessage>();
```